src/test/java/ToRemoveTest.java

```diff
@@ -0,0 +1,74 @@
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+
+import java.time.LocalDate;
+import java.util.Locale;
+
+import static org.junit.jupiter.api.Assertions.*;
+
+class ToRemoveTest {
+
+    @Test
+    @DisplayName("testToUpperCase")
+    void toUpperCase() {
+        final String TEST_STRING = "test";
+        String actualResult = ToRemove.toUpperCase(TEST_STRING);
+        String expectedResult = TEST_STRING.toUpperCase(Locale.ROOT);
+        assertEquals(expectedResult, actualResult, () -> "The upper case test failed");
+    }
+
+    @Test
+    @DisplayName("testMultiply")
+    void multiply() {
+        int a = 2;
+        long b = 3_000_000;
+        assertTrue(ToRemove.multiply(a, b) == a * b, () -> "Muliply test failed");
```

Comment:
No. Please _explicitly_ specify expected value. The same way you did in TestNG tests.